app/views/api/v2/hosts/show.json.rabl

```diff
@@ -25,3 +25,16 @@ end
 child :config_groups do
   extends "api/v2/config_groups/main"
 end
+
+host_additional_tabs(@host).each do |id, tab|
+  if tab.is_a? String
+    node do |host|
+      partial "api/v2/#{tab}", :object => host
+    end
+  else
+    child tab => id.to_sym do |aspect|
+      class_name = tab.class.name.underscore
+      extends "api/v2/#{class_name.pluralize}/base"
```

Comment:
hrmmm, all the current katello views are under  katello/api/v2/class_names/ (to follow rails module conventions), but this will look for them under api/v2/katello/class_names/.  Maybe we should provide a way to override this per aspect? 

 lets chat about this some more.